.changeset/long-moles-sneeze.md

```diff
@@ -0,0 +1,5 @@
+---
+"@hashicorp/design-system-components": patch
+---
+
+`BadgeCount`: update the color tokens to be more accurate token for the usage.
```

Comment:
Maybe change to:

"update the color tokens to use palette tokens."

To be more explicit and because longer term it would probably be preferable to use semantic tokens vs. referencing palette colors directly.

Reply:
Definitely! I struggled to come up with a good changelog